testsuite/fixtures/crates/hello_1.0.0_8cac0afd/alire.gpr

```diff
@@ -0,0 +1,6 @@
+aggregate project Alire is
```

Comment:
Likewise, the whole alire-generated files don’t seem necessary: are you okay with removing them?

Reply:
OK